roles/gui-tools/tasks/install-obs.yml

```diff
@@ -0,0 +1,20 @@
+- name: Check if OBS is already installed
+  ansible.builtin.stat:
```

Comment:
Stat for files, maybe we should switch for a "does this command exists" which probably exists too in ansible, but might be done in another PR later ! :)